student-work/chana-willinger/Api-Final Project/ReadMe.md

```diff
@@ -39,3 +39,7 @@ Acknowledgments:
 This exercise was borrowed and adapted by Alex Fiedler.
 
 This exercise was borrowed and adapted by Good Docs Project Templates.
+
```

Comment:
```suggestion
## Additional Topics
```

Reply:
Use this syntax for the Introduction heading 
but make it H1 (like this):
"# heading name"